src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketTokenCredentialsImpl/help-secret.html

```diff
@@ -1,7 +1,8 @@
 <div>
-    <p>This is the value of the token that was displayed when the <a
-            href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Bitbucket personal
-        access token</a> was created in Bitbucket Server. It allows users to automatically set up build triggers when
-        creating a Jenkins project. Importantly, for this to work the token must have project admin permissions.</p>
-    <p>Users will only be able to use the token to set up build triggers.</p>
+    <p><a href="https://confluence.atlassian.com/display/BitbucketServer/Personal+access+tokens">Personal access
+        token</a> are generated in Bitbucket Server. The token you add must also have project admin permissions
+        for the projects that you want Jenkins to build from.</p>
+
+    <p>Providing this token will allow your users to automatically set up build triggers when creating Jenkins jobs.
+        They won't be able to use it for anything else.</p>
```

Comment:
This is strictly speaking not true, it is also used for build-status posting. If we're happy to lie a little bit here I'm happy to leave it.

Reply:
Yup, @ctan-atlassian and I have discussed about this. We decided to keep it as it is now.